sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.Xunit.cs

```diff
@@ -4,7 +4,7 @@
     using FluentAssertions;
     using Xunit;
 
-    class Program
+    public class Program
```

Comment:
This class is exactly the same as the _XXX.XUnit.Legacy.cs_ above, except for the namespaces.